commands/invoke.go

```diff
@@ -94,3 +114,20 @@ func runInvoke(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func generateSignedHeader(message []byte, key string, headerName string) (string, error) {
+
+	if len(headerName) == 0 {
+		return "", fmt.Errorf("signed header must have a non-zero length")
+	}
+
+	hash := hmac.Sign(message, []byte(key))
+	signature := hex.EncodeToString(hash)
+	signedHeader := fmt.Sprintf(`%s=%s=%s`, headerName, "sha1", string(signature[:]))
+
+	return signedHeader, nil
+}
+
+func missingSignFlag(hdr string, key string) bool {
```

Comment:
What is hdr? Can we update to longer name?

Reply:
reverted to `header`